### ISSUE TITLE
ipv6 problem / connection fails after ipv6

### ISSUE THREAD

**Issue by DannyPIR** [openmediavault/openmediavault#279]
Openmediavault 4.1.17.-1 on Dell T3400 Local Network on Router AVM-Berlin Fritzbox 6490 Cable - DS-Lite Stack - only IPv6 to Internet. Local ipv4 - works fine.

if I switch additional ipv6 to dhcp. no connection to omv. never. dead. only monitor and keypad direkt on nas pc -> ect/network/interfaces edit with nano ... delete all ipv6 lines --> save --> restart --> webgui works --> deactivate ipv6. then it works again.

please fix! i want to connect only over ipv6 and dissable ipv4. How to? Help-text? 

IPv6 comes! ipv4 is dead!

**Comment by votdev** [comment]
You're right. Maybe you could help to identify the problem. Is the ``/etc/network/interfaces`` file incorrect?

**Comment by votdev** [comment]
> ect/nginx/sites-available/openmed~-webgui add following at end under listen 80;
> listen [::]:80;

This line is created if IPv6 is enabled on your system.

https://github.com/openmediavault/openmediavault/blob/4.x/deb/openmediavault/usr/share/openmediavault/mkconf/nginx.d/10webgui#L57
https://github.com/openmediavault/openmediavault/blob/4.x/deb/openmediavault/usr/share/openmediavault/mkconf/nginx.d/10webgui#L108
https://github.com/openmediavault/openmediavault/blob/4.x/deb/openmediavault/usr/share/openmediavault/scripts/helper-functions#L824

So i'm wondering if IPv6 was not enabled or the helper function has an error.
